tools/ci/restrict_file_types.py

```diff
@@ -0,0 +1,67 @@
+import glob
+import re
+import os
+import sys
+import time
+from collections import namedtuple
+
+Failure = namedtuple("Failure", ["filename", "lineno", "message"])
+
+RED = "\033[0;31m"
+GREEN = "\033[0;32m"
+BLUE = "\033[0;34m"
+NC = "\033[0m"  # No Color
+
+def print_error(message: str, filename: str, line_number: int):
+    if os.getenv("GITHUB_ACTIONS") == "true": # We're on github, output in a special format.
+        print(f"::error file={filename},line={line_number},title=Restricted Type in File::{filename}:{line_number}: {RED}{message}{NC}")
+    else:
+        print(f"{filename}:{line_number}: {RED}{message}{NC}")
+
+if __name__ == "__main__":
+    print("restrict_file_types started")
+
+    exit_code = 0
+    start = time.time()
+
+    dm_files = glob.glob("**/*.dm", recursive=True)
+
+    if len(sys.argv) > 1:
+        dm_files = [sys.argv[1]]
```

Comment:
```suggestion
        dm_files = sys.argv[1:]
```
So that you can specify multiple.